tortoise/queryset.py

```diff
@@ -479,7 +475,8 @@ def get(self, *args: Q, **kwargs: Any) -> QuerySetSingle[MODEL]:
         """
         queryset = self.filter(*args, **kwargs)
         queryset._limit = 2
-        queryset._get = True
+        queryset._single = True
+        queryset._raise_does_not_exist = True
```

Comment:
Should you not also set it for `get_or_none`?

Reply:
I think `get_or_none` do not raise `DoesNotExist`. it will return None

Reply:
Doh, yet, you are right. It should however raise an exception if more than one record gets returned. I got confused about that.